### DISCUSSION TITLE
Network analytics discussion

### DISCUSSION THREAD

**Discussion by bro-n-bro-0** [bro-n-bro/bro.app#89] [Ideas]
## 👋 Welcome!
 
Here we can discuss network analytics such as behavioral science, tokenomics, and activity. Do not hesitate to add some params and criteria you want to know about the network

**Comment by bro-n-bro-99** [comment]
- [ ] consensus mechanism - the rules and protocols governing the network's consensus mechanism (quorum %?)
- [ ] hard forks and protocol upgrades - the history and details of any major upgrades or changes to the network's protocol (?)
- [ ] average transaction fees - the average fees paid by users for including their transactions in the blockchain
- [ ] ibc transfers 
- [ ] ibc volume
- [ ] ibc active channels (?)
- [ ] transactions
- [ ] daily active users
- [x] apr
- [x] inflation
- [x] token price
- [x] token price change (24h)
- [x] market cap
- [x] active proposals
- [x] current token supply
- [x] bonded amount 
- [x] unbonded amount
- [x] total bonded ratio
- [x] community pool
- [x] last block height
- [x] unbound period
- [x] transaction per block
- [ ] transaction per hour (?)
- [ ] blocks per hour
- [ ] block rate
- [x] active validators number
- [ ] missed blocks by validators
- [ ] next network upgrade (?) - time, time left, block height
- [ ] token emission schedule / burn time / halving time (?)
